app/service/FileService.scala

```diff
@@ -38,6 +38,10 @@ class FileService @Inject()(connector: BindingTariffFilestoreConnector, messages
     connector.upload(f).map(toFileAttachment(f.ref.file.length))
   }
 
+  private def toFileAttachment(size: Long): FilestoreResponse => FileAttachment = {
+    r => FileAttachment(r.id, r.fileName, r.mimeType, size)
```

Comment:
What does `r` represent? I find it hard to guess what these shot variable names mean when I'm skim reading the code

Reply:
FilestoreResponse => FileAttachment   . I think is quite clear for one line code